app/controllers/v0/institutions_controller.rb

```diff
@@ -147,6 +141,12 @@ def embed(group_counts)
         array << { name: k, count: v }
       end
     end
+    #private method added to account for added approved column
```

Comment:
Nitpicks:

Probably don't need to comment that this is a private method, it's implicit.

Also ensure there's a line of whitespace between `private` and the next line of code.

Reply:
Comment and extra private statement was removed.